src/pages/guidelines/icons/usage.mdx

```diff
@@ -45,15 +45,15 @@ important information.
 <Column colLg={4} colMd={4}  noGutterSm>
   <ResourceCard
     subTitle="Get the IBM Icons (16px, 20px) library"
-    href="sketch://add-library/cloud/KW2yr"
+    href="sketch://add-library/cloud/028e0598-591e-428c-a490-f6ec64b15ea7"
     >
       <MdxIcon name="sketch" />
  </ResourceCard>
 </Column>
 <Column colLg={4} colMd={4}  noGutterSm>
   <ResourceCard
     subTitle="Get the IBM Icons (24px, 32px) library"
```

Comment:
```suggestion
    subTitle="IBM Icons (24px, 32px) library"
```

Reply:
@janchild Thank you.
Could you look at this page too? https://www.carbondesignsystem.com/designing/kits/sketch
There are icons, but also theme libraries there and these you "Get".

Reply:
@mariuszmickiewicz Thank you for pointing that out! We need to clean up that page too. I can open a PR for that. We have the verb in the introductory text above all of these, and so we don't need to repeat it on the tile. Even if we don't have introductory text though, including the verbs on the tiles just adds clutter.